pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -128,6 +127,12 @@ buildPythonPackage rec {
       "test_close"
     ];
 
+  disabledTestPaths = [
+    # Imports python-on-whales, which requires dbx (and thus pydantic_1)
```

Comment:
That comment is wrong. dbx depends through many, many packages on this one and has an override for pydantic_1 which breaks tests in python-on-whales.